management/server/http/posture_checks_handler.go

```diff
@@ -292,6 +296,21 @@ func validatePostureChecksUpdate(req api.PostureCheckUpdate) error {
 		}
 	}
 
+	if processCheck := req.Checks.ProcessCheck; processCheck != nil {
+		if len(processCheck.Processes) == 0 {
+			return status.Errorf(status.InvalidArgument, "processes for process check shouldn't be empty")
+		}
+
+		for _, process := range processCheck.Processes {
+			if process.WindowsPath == "" {
+				return status.Errorf(status.InvalidArgument, "windows path for process check shouldn't be empty")
```

Comment:
shouldn't this check be in account manager? and we should allow empty paths. We will have checks for one type of operating system, and if the path is empty, the check will fail as expected, not allowing access to the system.